Source/Fuse.Maps/MapView.uno

```diff
@@ -219,6 +240,16 @@ namespace Fuse.Controls
 			UpdateMarkersNextFrame();
 		}
 
+		void OnPolylineAdded(MapPolyline polyline)
+		{
+			UpdateMarkersNextFrame();
+		}
+
+		void OnPolylineRemoved(MapPolyline polyline)
+		{
+			UpdateMarkersNextFrame();
+		}
```

Comment:
I'm not sure polylines and markers share context here. Since markers are "uploaded" in a batch every time UpdateMarkersNextFrame is there to buffer marker changes before applying them in bulk. I'm not sure markers should update if the polylines have changed and vice versa?